src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
1. Duplicated `export const BASE_URL` see file `src/api/api.ts:1` 
2. Move reusable constant variables to separate file (e.g. - `/shared/shared.js`) and import them when needed